core/src/main/java/org/elasticsearch/cluster/routing/allocation/decider/Decision.java

```
@@ -302,6 +309,13 @@ public String label() {
         }
 
         @Override
+        @Nullable
+        public String getExplanation() {
+            // multi decisions have no explanation
+            return null;
```

should this throw an UnsupportedOperationException?